ui/webui/src/components/header.jsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with This program; If not, see <http://www.gnu.org/licenses/>.
+ */
+import cockpit from "cockpit";
+
+import React from "react";
+
+import {
+    Flex,
+    Label,
+    PageSection, PageSectionVariants,
+    Popover,
+    TextContent, Text
+} from "@patternfly/react-core";
+import { InfoCircleIcon } from "@patternfly/react-icons";
+
+const _ = cockpit.gettext;
+
+const prerelease = _("Pre-release");
+
+export const AnacondaHeader = (props) => {
+    let betanag;
+    if (props.beta) {
+        betanag = (
+            <Popover
+              headerContent={_("Fedora ## pre-release")}
+              bodyContent={_(`Notice: This is pre-released software that is intended for development and testing purposes only. Do *not* use this software for any critical work or for production environments.
```

Comment:
For allowing cockpit to translate strings you need to use double quote. Stupid limitation.

Reply:
A number of questions.
- Moving from backtick strings to double-quoted ones means I lose the ability to keep this a multi-line literal. How do I wrap it nicely? How do I keep it such that the translation extractor understands it as one literal, not three?
- Can I have 3 paragraphs instead of one? If so, can I make it somehow so that the `<p>` etc. are inside the string literal?